client/signup/jetpack-connect/index.jsx

```diff
@@ -199,6 +199,10 @@ class JetpackConnectMain extends Component {
 			return false;
 		}
 
+		if ( this.checkProperty( 'isJetpackConnected' ) ) {
```

Comment:
Is the reordering necessary? When working on #13500 I noticed that these strings which are passed to `JetpackConnectNotices` do not all match. See bb9e1e52b632d5845a0f819b829902faaa95868c

It may be a question of adding `alreadyOwned` to `JetpackConnectNotices`, I'm not sure if there's a condition `isJetpackConnected && ! userOwnsSite`.